apoiase/app.py

```diff
@@ -0,0 +1,30 @@
+import json
+
+import requests
+from flask import Flask, jsonify
+from flask_cors import CORS
+
+
+app = Flask(__name__)
+app.config['JSON_AS_ASCII'] = False
+CORS(app)
+
+@app.route('/')
+def home():
+    response = requests.get('https://apoia.se/api/v1/users/serenata?format=json')
```

Comment:
Can you parameterize `serenata` here? This would make the project generic enough for other projects crowdfunded via APOIA.se.